test/fixtures/api/swagger.yaml

```diff
@@ -294,6 +294,23 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /sources:
```

Comment:
Yeah, it's quite annoying. However, I would rather not require a HTTP request to run the tests. It's also somewhat important to keep it together so if we use a tool like `git-bisect` to find for a bug, the tests will always pass, even if the current `swagger.yaml` is totally different from what it was at the time the test was written.